src/Database/DBStructure.php

```diff
@@ -990,15 +990,13 @@ public static function definition() {
 		$database["group_member"] = array(
 				"fields" => array(
 						"id" => array("type" => "int(10) unsigned", "not null" => "1", "extra" => "auto_increment", "primary" => "1"),
-						"uid" => array("type" => "int(10) unsigned", "not null" => "1", "default" => "0", "relation" => array("user" => "uid")),
 						"gid" => array("type" => "int(10) unsigned", "not null" => "1", "default" => "0", "relation" => array("group" => "id")),
 						"contact-id" => array("type" => "int(10) unsigned", "not null" => "1", "default" => "0", "relation" => array("contact" => "id")),
 						),
 				"indexes" => array(
 						"PRIMARY" => array("id"),
 						"contactid" => array("contact-id"),
 						"gid_contactid" => array("gid", "contact-id"),
```

Comment:
Shouldn'r this one now getting the unique index?

Reply:
Yes, but we talked about deduping first, so I wasn't sure how to tackle that. The actual change should be getting rid of the `id` column altogether and add a `PRIMARY` index on `gid, contact-id`.

Reply:
Changing the primary would be a much huger job. So for the start I would prefer this ```UNIQUE```.

Reply:
Got it.